specs/error_state/ErrorCodeStore.md

```diff
@@ -1,41 +1,30 @@
 # ErrorCodeStore state
 
 ## Procedure
-`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and 
-`MaxCodeSizeExceeded`.
-this type of error only occurs when executing create(create,create2) op code or tx deploy 
-transaction(tx.to = null).
+`ErrorCodeStore` is for two original code store related errors: `CodeStoreOutOfGas` and `MaxCodeSizeExceeded`. This type of error only occurs when executing create(create,create2) op code or tx deploy transaction(tx.to = null).
 
 ### EVM behavior
-When handling a CREATE-kind transaction, Initial bytecode opcodes will run and current call context is created.
-The final bytecode opcodes to store for new contract is the `RETURN` opcode of init codes result.
+When handling a CREATE-kind transaction, Initial bytecode opcodes will run and current call context is created. The final bytecode opcodes to store for new contract is the `RETURN` opcode of init codes result.
```

Comment:
Similarly as above, it would be great if this text is a bit improved. It's difficult to understand (for example, what does it mean "initial byte code opcodes will run"?)

Reply:
fixed in [6ad3b04](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475/commits/6ad3b041f054e6bed0b1a292bcaacc6e95cd6ecf)